mongodb/src/main/java/com/yahoo/ycsb/db/MongoDbClient.java

```diff
@@ -96,6 +96,8 @@
   /** A singleton Mongo instance. */
   private static MongoClient mongoClient;
 
+  private static Boolean dropDBBeforeRun;
```

Comment:
Thanks for all your comments, that very nice and helpful. I've modified code. Thanks!